src/components/authentication/authentication.css

```diff
@@ -75,6 +75,12 @@
   transform: translateX(-50%);
 }
 
+.error {
+  color: red;
+  display: inline-block;
+  text-transform: capitalize;
+}
+
 @media screen and (min-width: 589px) {
```

Comment:
relative units could be used here.

Reply:
I prefer to use absolute units for media queries? Any advantages of using relative units for media queries?